### ISSUE TITLE
ctrl-Enter keyboard shortcut does not work after upgrading to >2.0.0

### ISSUE THREAD

**Issue by adams-family** [node-red/node-red#3309]
### Current Behavior

Pressing ⌘-Enter (Mac) does not deploy a flow. Pressing Shift-Enter and **then** ⌘-Enter does work.

This used to work as expected (just by pressing ⌘-Enter) before upgrading Node-Red to v2.0.

### Expected Behavior

Pressing ⌘-Enter should deploy the flow (given the keyboard shortcut configuration in the repro steps)

### Steps To Reproduce

1) Define keyboard shortcut for Deploy flows:

<img width="721" alt="image" src="https://user-images.githubusercontent.com/19559733/147405386-a907cd39-73eb-4546-8464-be82ca89d913.png">

<img width="709" alt="image" src="https://user-images.githubusercontent.com/19559733/147405434-bf8f2f31-d8e5-4a9d-b7c1-6b0ccd67fe3f.png">

2) Make a change in a flow

3) Press ⌘-Enter

### Example flow

It's independent on flow configuration.

### Environment

- Node-RED version: `2.1.3`
- Node.js version: `14.18.1`
- npm version: `6.14.15`
- Platform/OS: NodeRed in `Docker`, client on `MacOS`
- Browser: `Safari 15.0`

**Comment by knolleary** [comment]
`ctrl-enter` is used in the default keyboard shortcuts for the `core:confirm-edit-tray` action. Can you check to see if you have the bound as well? That could be causing a clash.

**Comment by Steve-Mcl** [comment]
@knolleary as of 1 year ago, <kbd>CTRL-Enter</kbd> was assigned to `"core:go-to-selection"` with a scope of `"red-ui-workspace"` 

I can confirm <kbd>CTRL-Enter</kbd> does deploy if the d3 chart is not selected (i.e. click on the sidebar first)
I can also confirm <kbd>CTRL-Enter</kbd> works everywhere if I remove the binding from `"core:go-to-selection"`

@adams-family please do the following...
1. clear the binding for  `"core:go-to-selection"` 
![image](https://user-images.githubusercontent.com/44235289/156883996-c8dc6e2d-a45e-4b7e-bc7f-da3eee98c36d.png)
2. add the deploy binding (re-add if necessary) 
![image](https://user-images.githubusercontent.com/44235289/156883984-f80ed129-d172-4998-83b0-3631e8b1ebef.png)
3. press Done
4. test

**Comment by Steve-Mcl** [comment]
@knolleary 2 issues i noticed while testing this:
1. Assigning  CTRL-Enter  while already assigned to another shortcut (with different scope) did work, but clearing the "core:go-to-selection" shortcut cleared the "Deploy" shortcut as well. 
2. Using a deploy shortcut is possible while the button is shaded (i.e. mode a node, open a nodes props, press the shortcut) triggers a deploy.

**Comment by Steve-Mcl** [comment]
Related #3473

**Comment by adams-family** [comment]
@knolleary Nice! Clearing "Go To Selection" did do the job. Thanks!

Just a side-note, when trying to clear the keyboard shortcut for "Go To Selection" I had to do it three times... I cleared it (with backspace), closed the window, did not work. I reloaded the page (F5) and the shortcut was back. Weird. I repeated the steps about 3 times, reloaded the page and then it worked. I'm not really aware of what caused it.
